README.md

```diff
@@ -19,9 +19,9 @@ The library is currenty able to convert the following pixel formats:
 
 | Source pixel format  | Destination pixel formats  |
 | -------------------- | -------------------------- |
-| ARGB                 | I420, NV12                 |
-| BGR                  | I420, NV12                 |
-| BGRA                 | I420, NV12                 |
+| ARGB                 | I420, NV12, I444           |
```

Comment:
Sort destination pixel format in lexicographic order

Reply:
fixed